src/pages/library/Code.js

```diff
@@ -26,7 +25,12 @@ const styles = {
   }
 }
 
-export const MethodCard = ({ method: { name, synopsis }, ...props }) => {
+export const MethodCard = ({ method, ...props }) => {
+  const isMethodsRepoMethod = method.namespace && method.name
+
+  const name = isMethodsRepoMethod ? method.name : method.toolname
+  const description = isMethodsRepoMethod ? method.synopsis : method.description
```

Comment:
Dockstore workflow descriptions contain Markdown, so it doesn't look great here. The Featured Workspaces page also has unrendered Markdown descriptions though.

Reply:
Did UX have any thoughts on it?

Reply:
I didn't ask. The Featured Workspaces is the same way, so I took that as adequate precedent.